### ISSUE TITLE
[Android] Custom notification icon

### ISSUE THREAD

**Issue by filfat** [homerours/cordova-music-controls-plugin#21]
The current icon is either a play or pause button depending on the playback status,
it would be great if we could set an optional custom icon (eg, the app icon).

**Comment by homerours** [comment]
Good idea. I'll try to look at this.

**Comment by cinder92** [comment]
+1

**Comment by djmMax** [comment]
:+1:

**Comment by simonmartinez** [comment]
Yes, i need it too :+1:

**Comment by shawns582** [comment]
Any progress on this?  I tried editing my local copy of MusicControlsNotification.java(~line 154) but it must be wrong because my app is crashing.  I could use some help from someone who knows android programming enough to fix it:

Instead of builder.setSmallIcon(Icon.createWithContentUri("file://img/logo_white"));

**Comment by shawns582** [comment]
Just figured it out:

builder.setSmallIcon(context.getResources().getIdentifier("ic_notification_white2", "drawable", context.getPackageName()));  

where my file is in the platforms/android/res/drawable folder and is named ic_notification_white2.png.

**Comment by Yusadolat** [comment]
@shawns582 @homerours  Please can this plugin works with HTML5 Audio Player??. If Yes How do I go About That?

Thanks in Advance

**Comment by dermansi** [comment]
+1

**Comment by shawns582** [comment]
@Yusadolat yes it does. Check out the readme. For the event that fires (ie. music-controls-pause), you can code it to call the pause() function for the HTML5 audio player.

**Comment by dermansi** [comment]
The provided java solution doesn't work for me. Seems like it's not updating my changes to MusicControlsNotification.java. Can someone help me here?

**Comment by CVeniamin** [comment]
I think that commit https://github.com/homerours/cordova-music-controls-plugin/commit/d71d882cb9dd0f9ca71c54198d9ea880e798b800 solves the issue by adding support for custom notification icon
